src/More/ExternalLink.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from "react"
+import { StyleSheet, TouchableOpacity, Linking } from "react-native"
+
+import { GlobalText } from "../components"
+import { Typography } from "../styles"
+
+type ExternalLinkProps = {
```

Comment:
What are your thoughts on using `interface` here for consistency? Based on what I see elsewhere in the app, when we type the props object of a `FunctionComponent` we use `interface`. Totally a stylistic preference but curious to get your thoughts!

Reply:
The idea for using types vs interfaces is that types don't merge declarations, for example if you have a type `A` defined here and a type `A` defined lower in the file, you will see an error from the compiler and if you do it with interfaces typescript will merge the declarations if it's possible. I don't see a strong reason to use type vs interfaces on any of these files, especially since none of those types are exported but since types are the stronger declaration I tend to reach for them. Also types can be used as "values" in some scenarios. No problem to change it back to an interface if we have been using interfaces.

Reply:
Makes sense! Yeah totally a personal preference, thanks for sharing, very helpful/educational :)